### DISCUSSION TITLE
Subscription consumption erroring on wrong node

### DISCUSSION THREAD

**Discussion by bmills74** [ravendb/ravendb#14657] [Q&A]
When consuming my subscription, I'm getting an error stating ' 'CalendarMatchSubscription' can't be processed on current node (A), because it belongs to C\n'.  In the help docs it says this might happen if there is no leader in the cluster, but I definitely do have one.
Any idea what I need to do please? (attached error log)

[subserror.txt](https://github.com/ravendb/ravendb/files/9232459/subserror.txt)

**Comment by ayende** [comment]
Your client connected to A, but the ownership of the subscription was moved to C.
The subscription handles that automatically. There should be nothing you have to do.

**Reply by bmills74** [reply]
problem is the subscriptions are not being activated because it's throwing the above error when I try run them.

![image](https://user-images.githubusercontent.com/8238067/182135141-6573f215-37c8-45c7-93c7-7cd1d238b17b.png)

**Reply by ayende** [reply]
All of them are marked as disabled.

**Reply by bmills74** [reply]
> All of them are marked as disabled.

How did I miss that!
 *hides under rock*
